Prace_domowe/Praca_domowa2/Tomek_Nocoń/WUMPD2.ipynb

```diff
@@ -0,0 +1,855 @@
+{
```

Comment:
<p>Trochę szkoda, że te dwa boxploty nie zostały umieszczone na jednym wykresie -- ułatwiłoby to porównywanie wyników. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>